ramlfications/utils/types.py

```diff
@@ -82,48 +84,52 @@ def parse_examples(raml_version, data):
     #
     data = data or {}
     kw = {}
-    example = data.get("example", None)
+    example = data.get("example")
 
     if raml_version == "0.8":
         if "examples" in data:
             del data["examples"]
-            # Should emit a lint warning if there's an examples node and
+            # TODO: Emit a lint warning if there's an examples node and
             # we're processing RAML 0.8: authors may be expecting it to
             # be honored.
         kw["example"] = example
 
-    if raml_version >= "1.0":
+    if raml_version != "0.8":
         if "example" in data:
             kw["example"] = parse_example(None, example)
 
+        kw["examples"] = None
         if "examples" in data:
             if "example" in data:
-                raise RuntimeError("example and examples cannot co-exist")
+                # TODO: Emit a lint warning during validation.
+                raise InvalidRAMLStructureError(
+                    "example and examples cannot co-exist")
```

Comment:
Nitpick: can you add single quotes around the properties for readability? e.g. `"'example' and 'examples' cannot co-exist"`